tests/rpc/conftest.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import datetime
+import os
+import pytest
+import tempfile
+from threading import Thread
+import time
+
+from evm.rpc.ipc import (
+    get_test_chain,
+    start,
+)
+
+
+@pytest.fixture(scope='session')
+def ipc_pipe():
+    tmpdir = tempfile.gettempdir()
+    unique_id = datetime.datetime.timestamp(datetime.datetime.now())
```

Comment:
Unless you were intentionally going for a timestamped ipc socket you could use `uuid.uuid4()` for this as well.